src/main/resources/static/_shared.yml

```diff
@@ -505,6 +505,16 @@ components:
       required:
         - arrivalDate
         - departureDate
+    WithdrawPlacementRequest:
+      type: object
+      properties:
+        reason:
+          type: string
+          enum:
+            - DuplicatePlacementRequest
+            - AlternativeProvisionIdentified
```

Comment:
Could we make this enum a named object? That way it'll be easier to use in the frontend

Reply:
Yep, i'll remember to do that for new enums going forward